webpack.config.js

```diff
@@ -74,7 +74,7 @@ module.exports = ({ mode }) => {
       devServer: {
         historyApiFallback: true,
         contentBase: path.join(__dirname, 'public'),
-        port: 3000,
+        port: 4000,
```

Comment:
What's the need to change the port?

Reply:
@anishagg17 Thanks for the quick review! I think it got added automatically because of the pre-commit hook. I have reverted it now. Should I squash the commits?